ruby/ql/lib/codeql/ruby/security/SensitiveActions.qll

```diff
@@ -0,0 +1,68 @@
+/**
+ * Provides classes and predicates for identifying sensitive data and methods for security.
+ *
+ * 'Sensitive' data in general is anything that should not be sent around in unencrypted form. This
+ * library tries to guess where sensitive data may either be stored in a variable or produced by a
+ * method.
```

Comment:
I guess that this file is based off the JS version of this library? Looks like it will be a good place to include heuristics for the existing `rb/hardcoded-credentials` query and for future queries about leaking sensitive data.

Reply:
Yes, it's based on the JS version.